install/ontology/blueprints.rdf

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0"?>
+<rdf:RDF
+        xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+        xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#"
+        xml:base="http://www.taotesting.com/ontologies/blueprints.rdf#"
+        xmlns:widget="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#"
+        xmlns:generis="http://www.tao.lu/Ontologies/generis.rdf#"
+        xmlns:tao="http://www.tao.lu/Ontologies/TAO.rdf#"
+>
+    <!-- Class to represent test blueprints -->
+    <!-- Contains identifer, title & content to store additional info -->
+    <rdf:Description rdf:about="http://www.taotesting.com/ontologies/blueprints.rdf#Blueprints">
+        <rdfs:label xml:lang="en-US"><![CDATA[Blueprints]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Blueprints constraints.]]></rdfs:comment>
+        <rdfs:subClassOf rdf:resource="http://www.tao.lu/Ontologies/TAO.rdf#TAOObject"/>
+    </rdf:Description>
+
+    <!-- Blueprints identifier -->
+    <!-- Not required -->
+    <!-- Not language dependent -->
+    <rdf:Description rdf:about="http://www.taotesting.com/ontologies/blueprints.rdf#identifier">
+        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[Blueprints identifier]]></rdfs:label>
+        <rdfs:comment xml:lang="en-US"><![CDATA[Blueprints identifier]]></rdfs:comment>
+        <rdfs:domain rdf:resource="http://www.taotesting.com/ontologies/blueprints.rdf#Blueprints"/>
+        <rdfs:range rdf:resource="http://www.w3.org/2000/01/rdf-schema#Literal"/>
+        <generis:is_language_dependent rdf:resource="http://www.tao.lu/Ontologies/generis.rdf#False"/>
+        <widget:widget rdf:resource="http://www.tao.lu/datatypes/WidgetDefinitions.rdf#TextBox"/>
+        <tao:TAOGUIOrder><![CDATA[1]]></tao:TAOGUIOrder>
+    </rdf:Description>
+
+    <!-- Blueprints title -->
+    <!-- Not required -->
+    <!-- Language dependent -->
+    <rdf:Description rdf:about="http://www.taotesting.com/ontologies/blueprints.rdf#title">
+        <rdf:type rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#Property"/>
+        <rdfs:label xml:lang="en-US"><![CDATA[Blueprints title]]></rdfs:label>
```

Comment:
Mhhhhh... now I'm thinking. Should'nt we use the label as the title? rdfs:label + blueprints:title seems to me redundant. As soon as we make sure there is an event based mechanism to synchronize the title in the blueprint in the authoring tool and the resource label, we are all good !